.github/workflows/definitelyTyped.yml

```diff
@@ -0,0 +1,55 @@
+name: DefinitelyTyped Tests
+
+on:
+  push:
+    branches:
+      - 'feature/definitelytype'
```

Comment:
just as a reminder. I believe you want to change it to the cron job.

Reply:
I'm still working on this file to make sure it will ignore the submodule, right now I have to keep the "on push" hook so that my tests runs everytime I push